README.md

```diff
@@ -1,17 +1,13 @@
-Faker
+factory-helper
 =====
-This gem is a port of Perl's Data::Faker library that generates fake data.
+This gem began as a fork of stympy's Faker which is a port of Perl's Data::Faker library that generates fake data.
```

Comment:
I'm happy with "This gem began life as a fork of [stympy's Faker](https://github.com/stympy/faker)."